### PR TITLE
Clean setup function in setup.py

### PR DESCRIPTION
**Pull request by Pahandrovich**

### CODE REVIEW

File: setup.py [hunk 1]
```diff
@@ -356,12 +356,22 @@ def gen_pyx(odir):
 
 gen_pyx(os.path.abspath('./build'))
 
+with open(os.path.abspath('./README.md')) as f:
+    LONG_DESCRIPTION = f.read()
```
Comment (napetrov) [inline setup.py:360]: guess it would be super long description - think we need less details. =)

File: setup.py [hunk 2]
```diff
@@ -372,13 +378,13 @@ def gen_pyx(odir):
...
             'Operating System :: Microsoft :: Windows',
             'Operating System :: POSIX :: Linux',
-            'Programming Language :: Python :: 2.7',
             'Programming Language :: Python :: 3',
             'Topic :: Scientific/Engineering',
             'Topic :: System',
             'Topic :: Software Development',
           ],
         setup_requires = ['numpy>=1.14', 'cython', 'jinja2'],
+        install_requires = ['numpy>=1.14', 'tbb', 'impi_rt','daal', 'dpcpp_cpp_rt', 'common_cmplr_lib_rt'],
```
Comment (PetrovKP) [inline setup.py:387]:
```suggestion
        install_requires = ['numpy>=1.14', 'tbb', 'impi_rt','daal', 'dpcpp_cpp_rt'],
```

**Review by PetrovKP — APPROVED** [review]

File: setup.py [hunk 3]
```diff
@@ -384,7 +384,7 @@ def gen_pyx(odir):
             'Topic :: Software Development',
           ],
         setup_requires = ['numpy>=1.14', 'cython', 'jinja2'],
-        install_requires = ['numpy>=1.14', 'tbb', 'daal', 'dpcpp_cpp_rt'] + ['impi_rt'] if not "osx" in sys.platform else [],
+        install_requires = ['numpy>=1.14', 'tbb', 'daal', 'dpcpp_cpp_rt'] + ['impi_rt'] if "osx" not in sys.platform else [],
```
Comment (PetrovKP) [inline setup.py:387]:
```suggestion
        install_requires = ['numpy>=1.14', 'tbb', 'daal', 'dpcpp_cpp_rt'] + ['impi_rt'] if not IS_MAC else [],
```

It seems to me more and more that getting rid of the RT dependency of the MPI looks like a nice and correct solution :)

**Review by PetrovKP — APPROVED** [review]

**Comment by Pahandrovich** [comment]
http://intel-ci.intel.com/eb3a093c-f04d-f18b-80bb-8cdcd4b723a1

**Review by PetrovKP — APPROVED** [review]

**Pull request merged**